src/app/user/profile/profile.page.html

```diff
@@ -1,14 +1,68 @@
-<ion-header [translucent]="true">
-  <ion-toolbar>
-    <ion-title>Profile</ion-title>
-  </ion-toolbar>
-</ion-header>
-
 <ion-content [fullscreen]="true">
-  <ion-header collapse="condense">
-    <ion-toolbar>
-      <ion-title size="large">Profile</ion-title>
-    </ion-toolbar>
-  </ion-header>
-  <app-explore-container name="Profile"></app-explore-container>
+  <app-header />
+  <div collapse="condense">
+    <div class="heading-title">
+      <h1>Профил</h1>
+      <p>
+        {{user.role === "regular" ? 'на потребител' : "на организатор"}}
+      </p>
+    </div>
+  </div>
+  <div class="container">
+    <form #userEditForm="ngForm" (submit)="onUserEdit(userEditForm)">
```

Comment:
This is only valid if the user is regular. What if an organizer ?

Reply:
I made the form inside with a check if it is a user or organizer.